specification/components/stu3/schemas/endpoints/a024-retrieve-advice-and-guidance.yaml

```diff
@@ -9,22 +9,29 @@ description: |
   The conversation can be retrieved via [[HYPERLINK_A025]].
 
   ## Supported security patterns
+  - Application-restricted access, unattended access
```

Comment:
Has it been queried why this mentions 'unattended access' and the others don't?

Reply:
Raised to BA and BA updated all 4.

Reply:
Does that mean the references on the existing endpoints should also be updated to be consistent? NHSD were very keen on consistency during the first phase of app restricted OAS changes

Reply:
I can see that in addition to the A&G endpoints, A004 has also bee updated with 'unattended access'. Should the other earlier app restricted endpoints also mention this?

Reply:
I had a TL review yesterday with Adam and Jack and also spoke to Justine. There are few changes introduced part of this change including "unattended access" and some text ordering on pre-requisites are to be done through a wider piece of work at the whole OAS documentation change.